cmd/livepeer/starter/starter.go

```diff
@@ -204,6 +205,7 @@ func DefaultLivepeerConfig() LivepeerConfig {
 	defaultMaxTotalEV := "20000000000000"
 	defaultDepositMultiplier := 1
 	defaultMaxPricePerUnit := "0"
+	defaultEnforceMaxPrice := false
```

Comment:
I think conceptually it would make more sense for the max price to be enforced by default (equivalent to having `true` here [1]). That because the non-enforcing behavior can be very misleading for independent node operators, being a behavior that only really makes sense for Studio right now that prefers to overpay than to deny service. On the Studio side, we can toggle the flag in our deployment easily enough.

[1] Even though we could just change the default to `true`, it is always cumbersome to have boolean flags that default to true since you can't disable them as easily as enabling (e.g. just `--enforce-max-price`), and sometimes you can't disable them at all (mist haha). So my suggestion is to have an "inverse" flag here, which actually means enabling the feature of ignoring the max price if no Os match. WDYT of `--ignore-max-price-if-needed`, defaulting to `false`, or sth like that?

Reply:
> I think conceptually it would make more sense for the max price to be enforced by default (equivalent to having `true` here [1]). That because the non-enforcing behavior can be very misleading for independent node operators, being a behavior that only really makes sense for Studio right now that prefers to overpay than to deny service. On the Studio side, we can toggle the flag in our deployment easily enough.
> 
> [1] Even though we could just change the default to `true`, it is always cumbersome to have boolean flags that default to true since you can't disable them as easily as enabling (e.g. just `--enforce-max-price`), and sometimes you can't disable them at all (mist haha). So my suggestion is to have an "inverse" flag here, which actually means enabling the feature of ignoring the max price if no Os match. WDYT of `--ignore-max-price-if-needed`, defaulting to `false`, or sth like that?

I'm ok with this. @rickstaa wdyt?

Reply:
I agree with Victor's reasoning here

Reply:
@thomshutt I already discussed this with @victorges async and I am onboard with the suggested changes. Feel free to merge with the suggested changes 👍. Thanks for creating this pull request.

Reply:
Updated. @thomshutt please approve the PR and I'll merge